netmaster/objApi/apiController.go

```diff
@@ -737,38 +737,40 @@ func (ac *APIController) TenantCreate(tenant *contivModel.Tenant) error {
 		return err
 	}
 
-	// Create private network for the tenant
-	err = contivModel.CreateNetwork(&contivModel.Network{
-		Key:         tenant.TenantName + ":" + "private",
-		IsPublic:    false,
-		IsPrivate:   true,
-		Encap:       "vxlan",
-		PktTag:      1001,
-		Subnet:      "10.1.0.0/16",
-		Gateway:     "10.1.254.254",
-		NetworkName: "private",
-		TenantName:  tenant.TenantName,
-	})
-	if err != nil {
-		log.Errorf("Error creating privateNet for tenant: %+v. Err: %v", tenant, err)
-		return err
-	}
+	if tenant.TenantName == "default" {
```

Comment:
oh, k, sgtm. thanks @shaleman

Is this supposed to be checked in then, if it was just needed for @DivyaVavili testing? Either way is fine with me as you also seem to be working on changing this logic.